demo/Main.elm

```diff
@@ -23,20 +22,18 @@ main =
 -- MODEL
 
 
-type alias Data =
-    Service.ListQueuesResult
-
-
 type alias Model =
-    { response : Maybe (AWS.Response Data)
+    { queueUrls : List String
     , err : Maybe Http.Error
     }
 
 
 init : ( Model, Cmd Msg )
 init =
-    ( Model Nothing Nothing
-    , Task.perform GotDate Date.now
+    ( Model [] Nothing
+    , SQS.listQueues AWS.defaultOptions
+        |> AWS.toTask (SQS.config Config.region) creds
+        |> Task.attempt ListQueues
```

Comment:
I thought about calling it `send`. Or maybe `signAndSend`. Not sure. I chose `toTask` because `Http` module has a function with that name that does the same thing. @ngmiller what do you think?